tests/e2e/specs/analysis.cy.js

```diff
@@ -61,13 +61,16 @@ describe('Analysis view', () => {
           .get('td')
           .contains('eventually')
           .should('be.visible')
-        cy
-          .get('#c-analysis-filter-task-name')
-          .type('wait')
         cy
           .get('td')
           .contains('waiting')
           .should('be.visible')
+        cy
+          .get('#c-analysis-filter-task-name')
+          .click()
+          .get('.v-list-item')
+          .contains('waiting')
+          .click({ force: true })
```

Comment:
This has come underneath the check that `waiting` is visible in the table. It ought to be before it

Reply:
Done